modules/oe_theme_content_event/src/Plugin/ExtraField/Display/TeaserDetailsExtraField.php

```diff
@@ -42,6 +42,9 @@ public function viewElements(ContentEntityInterface $entity) {
 
     $this->addRenderableLocation($build, $entity);
     $this->addRenderableOnlineType($build, $entity);
+    // Override default icon size.
+    $build['#fields']['items'][0]['size'] = 'xs';
```

Comment:
Since Venue and Online type are optional fields, $build['#fields']['items'] can be empty. It has no sense to set icon size for such case.

```
  foreach ($build['#fields']['items'] as $key => $item) {
    if (!empty($build['#fields']['items'][$key]['size'])) {
      $build['#fields']['items'][$key]['size'] = 'xs';
    }
  }
```